oag/ja-jp/content/topics/access-gateway/boot-strap-oag-aws-post-deploy.htm

```diff
@@ -392,7 +392,7 @@ <h2><a name="Specify"></a>ホスト名を指定 </h2>
 パスワード： &lt;default-password&gt;</span>
                                                                     <p><span class="mc-variable okta-feature-names.Access_Gateway_CMD variable">Access Gateway 管理者コンソール</span>に初めてサインインする場合、デフォルトパスワードを変更する必要があります。<a href="admin-first-login-mgmt-console.htm" class="MCXref xref">「Access Gateway 管理者コンソールに初めてサインインする」</a>をご参照ください。 </p>
                                                                 </li>
-                                                                <li value="2"><span class="userinput">[5 - System（システム）</span>を入力します。<br /></li>
+                                                                <li value="2"><span class="userinput">[5 - システム</span>を入力します。<br /></li>
```

Comment:
@pengyao-okta Just curious why the translator removed the EN portion of the string here, but did not do the same on the line immediately following? We should be consistent.
Anyway, this is a `userinput` attribute, I don't think we provided specific bilingual guidance for this attr.